src/webui/src/components/public-child/OpenRow.tsx

```diff
@@ -105,7 +111,23 @@ class OpenRow extends React.Component<OpenRowProps, OpenRowState> {
                                         logCollection={EXPERIMENT.logCollectionEnabled}
                                     />
                                     :
-                                    <TrialLog logStr={logPathRow} id={trialId} />
+                                    <div>
+                                        <TrialLog logStr={logPathRow} id={trialId} />
+                                        {/* view each trial log in drawer*/}
+                                        <div id="trialog">
+                                            <div className="copy" style={{ marginTop: 15 }}>
+                                                <PrimaryButton
+                                                    onClick={this.openTrialLog.bind(this, 'TRIAL_LOG')}
+                                                    text="View trial log"
+                                                />
+                                                <PrimaryButton
+                                                    onClick={this.openTrialLog.bind(this, 'TRIAL_STDERR')}
+                                                    text="View stderr log"
+                                                    styles={{ root: { marginLeft: 15 } }}
+                                                />
+                                            </div>
+                                        </div>
+                                    </div>
```

Comment:
It seems the `MethodNotImplementedError` not handled by UI,  consider following options:
1.  the open log buttons only enabled for `local`
2.  show a proper error message when MethodNotImplementedError is encountered.